seatunnel-connectors-v2/connector-fake/src/main/java/org/apache/seatunnel/connectors/seatunnel/fake/source/FakeSourceReader.java

```diff
@@ -70,11 +70,8 @@ public void pollNext(Collector<SeaTunnelRow> output) throws InterruptedException
         synchronized (output.getCheckpointLock()) {
             FakeSourceSplit split = splits.poll();
             if (null != split) {
-                // Generate a random number of rows to emit.
-                List<SeaTunnelRow> seaTunnelRows = fakeDataGenerator.generateFakedRows(split.getRowNum());
```

Comment:
remove unused method `generateFakedRows`

Reply:
@zhilinli123 here

Reply:
> remove unused method `generateFakedRows`

The generateFakedRows method is used for other test cases

Reply:
Thank you for your help. I will submit it again

Reply:
CICD had been starting. Let's waiting CICD.

Reply:
@hailin0 Can you help me locate the reason why my CICD failed